src/apps/loan-list/utils/pagination.tsx

```diff
@@ -1,37 +1,34 @@
 import React, { useEffect, useState, FC } from "react";
-import { LoanV2 } from "../../../core/fbs/model/loanV2";
 import { getStackedItems, getListItems } from "./helpers";
 import LoanListItems from "../list/loan-list-items";
 import { GetMaterialManifestationQuery } from "../../../core/dbc-gateway/generated/graphql";
-import { LoanDetailsV2 } from "../../../core/fbs/model";
 import { ListView } from "../../../core/utils/types/list-view";
 import usePager from "../../../components/result-pager/use-pager";
 import { getPageSizeFromConfiguration } from "../../../core/utils/helpers/general";
+import { LoanMetaDataType } from "../../../core/utils/helpers/LoanMetaDataType";
```

Comment:
We have a directory especially for types, please use that:
src/core/utils/types

Reply:
I think this was an accident, the two folders are adjacent :)